docs/rules/index.md

```diff
@@ -4,108 +4,106 @@ sidebarDepth: 0
 
 # Available Rules
 
-The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) automatically fixes problems reported by rules which have a wrench :wrench: below.  
-The rules with the following star :star: are included in the `plugin:regexp/recommended` config.
+<!-- begin auto-generated rules list -->
 
-<!-- This file is automatically generated in tools/update-docs-rules-index.js, do not change! -->
+💼 Configurations enabled in.\
+⚠️ Configurations set to warn in.\
+✅ Set in the `recommended` configuration.\
+🔧 Automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/user-guide/command-line-interface#--fix).\
+💡 Manually fixable by [editor suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).\
+❌ Deprecated.
 
-### Possible Errors
+## Best Practices
```

Comment:
We're using `--split-by meta.docs.category` [option](https://github.com/bmish/eslint-doc-generator#configuration-options) to achieve this. The sub-lists are ordered alphabetically. I'm not sure how we could achieve the original ordering using a generic option (let me know if you have ideas).

Reply:
Hmm, categories are free for plugins, so I think it's hard to decide on a general sort. I think the only way to solve the requirement is to specify the compare function in the config.

Reply:
Good idea, filed here: https://github.com/bmish/eslint-doc-generator/issues/294

Reply:
Another option would be to allow users to specify an array of categories (e.g. `categorySort: ["Possible Errors", "Best Paractices", "Stylistic issues"]`). Categories not specified by the user could come in alphabetical order after the specified ones.

This would be a more declarative and JSON-compatible solution.

Reply:
I have solved this by defining a `ruleListSplit` function in the config file using the new feature of the [`--rule-list-split` config option](https://github.com/bmish/eslint-doc-generator#configuration-options) from eslint-doc-generator [1.2.0](https://github.com/bmish/eslint-doc-generator/releases/tag/v1.2.0). The new sections should now match the old sections.